docs/index.rst

```diff
@@ -20,10 +20,12 @@ and admin password were available, and, if and only if that file was changed,
 the appropriate service would be restarted:
 
 .. code-block:: python
+    from charms.reactive import when, when_file_changed
+    from charmhelpers.core import templating, hookenv
```

Comment:
Good catch! I missed that in my PR (which I'm going to withdraw, since you covered the doc change.